app.js

```diff
@@ -30,36 +28,51 @@ global.ADMIN_COMMANDS = require('./commands/admin.js');
 global.MOD_COMMANDS = require('./commands/mod.js');
 global.welcomeImage = "https://s26.postimg.cc/8x8hnunux/Reddit_Link.png";
 
+var winston = require('winston');
+
+global.logger = winston.createLogger({
+  level: 'debug',
+  format: winston.format.simple(),
+  transports: [
+    new winston.transports.Console({ level: 'info' }),
+    new winston.transports.File({ filename: 'error.log', level: 'error' }),
+    new winston.transports.File({ filename: 'info.log', level: 'info' }),
+    new winston.transports.File({ filename: 'debug.log' })
+  ]
+});
+
+logger = global.logger;
+
 client.on(`ready`, () => {
-  console.log(`I am ready!`);
+  logger.info(`I am ready!`);
   global.server = global.client.guilds.get(process.env.SERVER_ID);
   if(enableDB){
     global.util.updateUserPermissions();
     global.util.updateSpecialChannels();
-    console.log("Connected to database succesfully");
+    logger.info("Connected to database succesfully");
```

Comment:
Is it possible to add some additional info about what DB it is connected to and how? Using the env variables print out some of the info.